docs/Quickstart.mdx

```diff
@@ -42,67 +47,79 @@ The <div className="button gray_button"><UserOutlined /> Sign In</div> button sh
 </div>
 
-### Step 4: Add nodes to ingest and process data
-
-Click on the <div className="button darkblue_button">**Node** <PlusCircleOutlined /></div> button in the header bar and search for the term "excel" to filter for the relevant <NodeChip text="Excel_Read" /> *node*.  On the Flow diagram, select the <NodeChip text="Excel_Read" /> *node* and connect its top attachment point to the bottom attachment point of the <NodeChip text="Flow_Inputs" /> *node*.
+After a few seconds, your flow will be created and you'll be redirected to a blank flow editor page where you can edit your new flow.
 
+### Step 4: Add nodes to ingest and process data
 
+Click on the <div className="button darkblue_button">**Node** <PlusCircleOutlined /></div> button in the header bar and search for the term "excel" to filter for the relevant <div className="button darkblue_button">**Excel_Read**</div> node.  Click the <div className="button darkblue_button">**Add**</div> button to add it to your flow. 
 <div class="text--center">
-<img width="350" alt="Excel_Read node chip" src="https://ganymede-bio.mo.cloudinary.net/apiServer/QuickstartExcelRead_20230111.png"/>
+<img alt="Excel_Read node chip" src="https://ganymede-bio.mo.cloudinary.net/apiServer/QuickstartAddExcelNode20230501.png"/>
 </div> 
 
-Repeat this process with the <NodeChip text="Transform_Py" /> *node*.  Upon doing so, the <div className="button darkblue_button"><BranchesOutlined/> **Flow Editor**</div> window should look as follows:
+Then click the **X** button in the upper right corner to return to your flow, where you should now see an <div className="button darkblue_button">**Excel_Read**</div> node.
+<!-- TODO: This last clause should remain in docs until adding a node gives some feedback -->
+
+Click on the downward-pointing carat <DownOutlined className="button darkblue_button_icon"/> in the corner of the <div className="button darkblue_button">**Excel_Read**</div> node to expand it and edit its attributes.
 
-<img alt="Quickstart Flow Init" src="https://ganymede-bio.mo.cloudinary.net/apiServer/QuickstartFlow_20230405.png"/>
+<div class="text--center">
+<img alt="Quickstart flow description" src="https://ganymede-bio.mo.cloudinary.net/apiServer/QuickstartAddedNodeExcel2_20230501.png"/>
+</div>
 
-Node attributes with <div className="green_text">**green labels**</div> indicate validation attributes.  In this example, the <div className="green_text">**excel**</div> node attribute on <NodeChip text="Excel_Read" /> refers to the file extension that the *flow* looks for during runs.  Click on the edit button, and modify the <div className="green_text">**excel**</div> node attribute to be ".xlsx", so that only files with ".xlsx" as an extension would be let through.  
+Change the **excel** node attribute from "excel" to ".xlsx" so that only files with `.xlsx` as an extension will be processed by this node (other file types, such as `.xlsm` will be blocked). 
```

Comment:
Below this, we should add a note about changing the "results" parameter to Quickstart_Absorbance_Change_Excel_Read_results, which is mentioned later in the SQL query.

Reply:
yeah just tested again, this field auto-populates correctly, assuming the user named the flow "Quickstart..." as described.